### DISCUSSION TITLE
Pi Zero with HaxoPi stuck in endless reboot

### DISCUSSION THREAD

**Discussion by pappavis** [cardonabits/haxo-rs#47] [General]
My haxophone + Pi Zero gets to 'Reached sound card'

I bought a nw Pi Zero online with pre-soldered headers

Steps followed;
1. After receiving my Haxophone
2. Flashed Haxo-rpiz to a 128Gb SD-card, as I had one available
3. Mounted the Pi into the Haxophone' headerpins.
4. Connected a USB keybord adapter, HDMI for video and booted up.
5. Being the original Pi Zero booting is slow.
6. After <>3 minutes of booting the screen output says: 
  'Reached sound card'
 login:

7. When i type 'pi' <enter> a message is displayed: The system is still booting up.
8. After a few seconds the Pi reboots.
9. Rince and repeat from step 6. Endless reboot.

I have zero experience with Saxophones, and kinda proficient with Raspberry Pi's & ESP32s etc.

What did i do wrong?

**Comment by jcard0na** [comment]
Hi @pappavis,

The steps you outline seem correct to me.  Did you customize the haxo-rpiz image?  That is the only cause I know that[ can trigger automatic reboots](https://github.com/cardonabits/haxo-hw/discussions/59#discussioncomment-8039696).

Cheers,

**Comment by pappavis** [comment]
SOLVED.  My Haxophone is working: playing audio out via the 3.5mm out and MIDI on an original Pi Zero WH.

Steps;
1. I burnt haxo-rpiz  again to a SDCard with Raspberry Pi Imager.. with option: do not apply my custom settings.
2. Attached a USB cable on the inner USB-port & HDMI to the Pi.
3. Plugged in the USB.. waited a minute for it to bootup., to get to login: prompt.
4. Plugged in a 3.5mm microfone to the Haxophone' audioport
5. Started blowing on it & press some keys. Result: SOUNDS of a sax :))))
6. Started Logic Pro 11 (LP11), adding a software Midi
7. In LP11 in the Tracks, selected Midi Input = Haxophone and added a Midi region.
8. Using LP11 started blowing on it, pressed some keys.. and heard a Alto Sax.

Nice :))

Minor gripe;
The Pi during boot says: failed to start dhys-swapfile

I have 0 saxophone experience... Now I can get started :)

Below: My Haxophone in MIDI connected in Logic Pro 11
![image](https://github.com/user-attachments/assets/a64cc5f8-b607-4263-8a21-196b721dbfda)

**Comment by jcard0na** [comment]
Excellent.  Thanks for the detailed update.
